src/Tasks/Microsoft.NET.Build.Tasks/targets/GenerateDeps/GenerateDeps.proj

```diff
@@ -50,6 +50,10 @@ Copyright (c) .NET Foundation. All rights reserved.
          So we need to disable the logic which checks that the version of Microsoft.NETCore.App from the assets file
          matches the one that would be resolved based on the TargetFramework that was passed in. -->
     <VerifyMatchingImplicitPackageVersion>false</VerifyMatchingImplicitPackageVersion>
+
+    <!--In 3.0 and above, disable implicit framework references so we don't resolve targeting packs and conflict resolve 
```

Comment:
I would remove "In 3.0 and above" since it makes me think we need a condition here, but we don't.